devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy

```diff
@@ -82,6 +82,7 @@ def installed() {
 	state.temperatureSensorDni = null
 	sendHubCommand(encap(zwave.configurationV2.configurationGet(parameterNumber: 71)))
 	sendEvent(name: "checkInterval", value: 2 * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	schedule(new Date(), getEnergyUsage)
```

Comment:
Any reason you're using `schedule` here and not `runIn` or just sending the hub command from within here?

Reply:
I want it to run every day, as I've described in the ticket and according to [the documentation]([RW General][p.kacprowicz][2020.08.27] work start) using `schedule` with `new Date()` is one way to do it.

Reply:
huh, interesting